easybuild/easyconfigs/i/icc/README.md

```diff
@@ -0,0 +1,14 @@
+In order to use these configs you must first download the Intel Parallel Studio XE CPP file. NOT the full bundle! The file should look like this: parallel_studio_xe_2018_update2_composer_edition_for_cpp.tgz    
+
+Next you must specify the license file. There are four options:  
+* Place it in the following path: ~/licenses/intel/license.lic
+* Set an environment variable: `export INTEL_LICENSE_FILE=/path/to/file.lic`
+* Set an environment variable: `export LM_LICENSE_FILE=/path/to/file.lic`
+* Edit the EB script and modify or add the line `license_file = license_file`
```

Comment:
should probably formatting like (or something like it)?

```
`license_file = "/path/to/licenses/intel.lic"`
```

Reply:
Good catch. Changing now.